### DISCUSSION TITLE
Font Color Change

### DISCUSSION THREAD

**Discussion by meghanaviyyapu** [pymupdf/PyMuPDF#2467] [Looking for help] [answered]
Can font color of text be changed using PyMuPDF?

**Comment by JorjMcKie** [comment]
There is no direct way. You can use redaction annotations of course.

**Reply by meghanaviyyapu** [reply]
Could you please share any code examples
[sample.pdf](https://github.com/pymupdf/PyMuPDF/files/11733839/sample.pdf)
In this PDF, the first heading has to be modified to a different color. Could you let me know how it can be done?

**Reply by JorjMcKie** [reply, marked as answer]
Here is a simple script: 
[sample.zip](https://github.com/pymupdf/PyMuPDF/files/11734154/sample.zip)

**Reply by meghanaviyyapu** [reply]
Thank you
